vendor.conf

```diff
@@ -13,7 +13,7 @@ github.com/golang/protobuf 8ee79997227bf9b34611aee7946ae64735e6fd93
 github.com/opencontainers/runc ce450bcc6c135cae93ee2a99d41a308c179ff6dc
 github.com/opencontainers/runtime-spec v1.0.0-rc3
 github.com/Sirupsen/logrus v0.11.0
-github.com/stevvooe/go-btrfs 8539a1d04898663b8eda14982e24b74e7a12388e
+github.com/stevvooe/go-btrfs 6de97c518fded4c706e928aa93922bc5c1809687 https://github.com/AkihiroSuda/go-btrfs.git
```

Comment:
I'll vendor the upstream repo when https://github.com/stevvooe/go-btrfs/pull/6 gets merged